openshift/topics/advanced_concepts.adoc

```diff
@@ -594,6 +594,49 @@ There are two methods for passing the {project_name} adapter configuration to th
 
 See xref:Example-EAP-Manual[Example Workflow: Manually Configure an Application to Use {project_name} Authentication, Using SAML Client] for an end-to-end example of the manual {project_name} client registration method using a SAML client.
 
+=== Using {project_name} vault with Openshift secrets
+. Specify a directory for the vault using the *_SSO_VAULT_DIR_* environment variable. Add following snippets to your template: (Alternatively you can add *_SSO_VAULT_DIR_* variable to the environment in your deployment's configuration)
```

Comment:
```suggestion
. Specify a directory for the vault using the *_SSO_VAULT_DIR_* environment variable. You can introduce the *_SSO_VAULT_DIR_* environment variable directly in the environment in your deployment configuration. It can also be included in the template by addding thew following snippets at the appropriate places in the template:
```